src/components/message/message.test.tsx

```diff
@@ -0,0 +1,158 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Message from "./message.component";
+import I18nProvider from "../i18n-provider";
+import enGB from "../../locales/en-gb";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+
+test("renders with provided children", () => {
+  render(<Message>Message</Message>);
+
+  expect(screen.getByText("Message")).toBeVisible();
+});
+
+test("renders with provided `title`", () => {
+  render(<Message title="Title">Message</Message>);
+
+  expect(screen.getByText("Title")).toBeVisible();
+});
+
+test("does not render component when `open` prop is false", () => {
+  render(
+    <Message data-role="my-message" open={false}>
+      Message
+    </Message>
+  );
+
+  expect(screen.queryByTestId("my-message")).not.toBeInTheDocument();
+});
+
+test("renders with close button if `onDismiss` is provided and calls onDismiss when clicked", async () => {
```

Comment:
**suggestion** (non-blocking): feel free to separate these in two tests, as we're testing two different things here. If it fails, it may cause a little bit of extra effort to debug